android/PhysicalWeb/app/src/main/java/org/physical_web/demos/WifiDirectHelloWorld.java

```diff
@@ -0,0 +1,55 @@
+package org.physical_web.demos;
+
+import org.physical_web.physicalweb.FileBroadcastService;
+import org.physical_web.physicalweb.R;
+
+import android.content.ContentResolver;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+
+/**
+ * Hello world demo for WifiDirect.
+ */
+public class WifiDirectHelloWorld implements Demo {
+  private static final String TAG = WifiDirectHelloWorld.class.getSimpleName();
+  private static boolean mIsDemoStarted = false;
+  private Context mContext;
+
+  public WifiDirectHelloWorld(Context context) {
+    mContext = context;
+  }
+
+  @Override
+  public String getSummary() {
+    return mContext.getString(R.string.wifi_direct_demo_summary);
+  }
+
+  @Override
+  public String getTitle() {
+    return mContext.getString(R.string.wifi_direct_demo_title);
+  }
+
+  @Override
+  public boolean isDemoStarted() {
+    return mIsDemoStarted;
+  }
+
+  @Override
+  public void startDemo() {
+    Intent intent = new Intent(mContext, FileBroadcastService.class);
+    Uri uri = Uri.parse(ContentResolver.SCHEME_ANDROID_RESOURCE + "://" +
+        mContext.getPackageName() + "/" + R.raw.wifi_direct_default_webpage);
+    intent.putExtra(FileBroadcastService.FILE_KEY, uri.toString());
```

Comment:
Would it make sense to use the string directly instead of converting it to a Uri and back?